Khan/wonder-blocks
# Use solid underline for links (until audit can be completed)

## Summary:
While the new dashed link is available, we are updating it to the old underline
style to match the non-wonder blocks links across webapp. An audit is currently
being run so we can update all these to wonder blocks link.

Once all the legacy links have been updated to use Wonder Blocks Link, we can
finally use the dashed link style.

- Update styles so that the links use solid 1px underlines.
- Comment out the old styles and add a TODO comment above them so we can easily go back.
  - The TODOs should reference WB-1521.

Issue: https://khanacademy.atlassian.net/browse/WB-1520

## Test plan:
- Go to http://localhost:53078/?path=/story/link--variants
- Confirm all the chromatic interaction tests are passing
- Confirm all the links have 1px solid underlines with no offset.
